requirements_docs.txt

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2015-2023, Blue Brain Project/EPFL
+#
+# All rights reserved.
```

Comment:
Is the copyright needed for the list of requirements as well?

Reply:
I took BluePyOpt for reference, and there there also was a Copyright header